.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -271,16 +169,27 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
-      - name: (Artifact re-build) Build Webpack
-        if: steps.download_artifact_1.outcome == 'failure'
-        run: python -m scripts.build --prod_env
-      - uses: ./.github/actions/install-oppia-dependencies
+      - name: Install Python deps
+        run: |
+          pip install --require-hashes --no-deps -r requirements.txt
+          pip install --require-hashes --no-deps -r requirements_dev.txt
```

Comment:
I thought we were using Docker?

Reply:
we need to install chrome on Github runner, so for that we maintain a python script to execute. thats why we need to install the python deps also

Reply:
added

Reply:
Can you share which packages have to be installed for the chrome script?

Reply:
https://github.com/oppia/oppia/blob/develop/scripts/install_chrome_for_ci.py#L22-L26
https://github.com/oppia/oppia/blob/develop/scripts/common.py#L17-L47
https://github.com/oppia/oppia/blob/develop/core/feconf.py#L19-L28
please refer here

Reply:
working on adding a bash script, and checking if it works fine. will update the below thread

Reply:
please continue discussion from this [thread](https://github.com/oppia/oppia/pull/18999/files#r1382870465)